packages/KBadge/KBadge.vue

```diff
@@ -49,14 +49,17 @@ export default {
 @import '~@kongponents/styles/_variables.scss';
 
 .k-badge {
-  display: inline-flex;
+  display: inline-block;
   justify-content: center;
   font-weight: 500;
+  vertical-align: top;
   font-size: var(--KBadgeFontSize, var(--type-sm, type(sm)));
-  height: 27px;
-  align-items: center;
-  width: var(--KBadgeWidth, 71px);
-  padding: var(--KBadgePaddingY, 2px) var(--KBadgePaddingX, 4px);
+  height: auto;
+  text-align: center;
+  min-width: var(--KBadgeMinWidth, 71px);
```

Comment:
So we even need a min width/height? I feel it should be set automatically by padding. And then max width to keep it from growing huge

Reply:
there is no min height (heigh auto could possibly be removed?). I added a min-width so that the immunity alert badge styles wouldn't regress.